Update Prerequisites Section (#2129)
Cxbx-Reloaded/Cxbx-Reloaded: README.md

  * OS: Windows 7+ x64, or x86-64 Linux with Wine. 32-bit is not supported.
  * GPU: Direct3D 9.0c with Pixel Shader Model 2.x, and Vertex Shader Model 3.0.

## Prerequisites
### Windows
  * [32-bit (x86) Visual C++ 2019 Redistributable](https://aka.ms/vs/16/release/vc_redist.x86.exe)
  * [Npcap *(used for network emulation)*](https://nmap.org/npcap/#download)
    * Make sure to enable winpcap compatibility mode!

> ### Wine

Reviewer: Above specifies the operating system Windows, so this line should probably be "Linux, using Wine" or something like that.

Author: I don't exactly agree with you. I consider Wine as its own platform since it can be use on Linux, BSD, Solaris, Android, and macOS. Maybe more that I missed.

Author: I do understand what you're trying to say. Perhaps you meant "Unix, using Wine"?

Author: Base on internal discussion, it's better to leave it as is. Since there's some variant of operating system which do not fall under Unix, such as Linux. And Wine isn't available for all POSIX operating system.

Since main developers are not on Linux full time, I forgot to add one detail under Wine section.

> **NOTICE: Please use latest stable release version of Wine. If it does not work for you, then roll back to Wine 5.0.3 as last known working version.**

Reviewer 2: ```suggestion
**NOTICE: Please use the latest stable release version of Wine. If it does not work for you, then roll back to Wine 5.0.3 which is the last known working version.**
```

Author: done

  * Winetricks
    * `vcrun2017` or `vcrun2019`

>       * NOTE: vcrun2019 require latest winetricks script!

Reviewer 2: ```suggestion
      * NOTE: vcrun2019 requires the latest winetricks script!
```

Author: done

    * `d3dcompiler_47`

>       * NOTE: May will change over time.

Reviewer 2: ```suggestion
      * NOTE: May/will change over time.
```

Author: Or better yet... **May be subject to** change over time.

Author: done

>   * Winpcap is built-in, no installation requirement.

Reviewer 2: ```suggestion
  * Winpcap is built-in, no installation is required.
```

Author: done
